### PR TITLE
Add travis CI

### PR DESCRIPTION
**Pull request by lopuhin**
Not including mypy check yet due to lxml typeshed issues https://github.com/python/typeshed/issues/525 - I'd rather add mypy support in a separate PR

### CODE REVIEW

**Comment by lopuhin** [comment]
@kmike this is ready for review - I'll add mypy in the next PR. There is only one tox env, but we might have more in the future.

**Comment by kmike** [comment]
Awesome! 

What do you think about adopting the same codecov config as in https://github.com/TeamHG-Memex/eli5/blob/master/codecov.yml?

**Comment by lopuhin** [comment]
Right, let's add it :)

Btw, what does ``project: false`` mean there - it must turn off the "project" check, is it the check that measures overall project coverage before and after?

**Comment by kmike** [comment]
Yes, it disables this check. For example, if a patch has 100% coverage, but it deletes more code than it adds, then project check could fail, because average coverage % could decrease.

**Comment by lopuhin** [comment]
Aha, thanks! Yes, that metrics always seemed a bit strange to track in such a way.
